lte/gateway/c/core/oai/include/mme_app_ue_context.h

```diff
@@ -456,6 +456,8 @@ typedef struct ue_mm_context_s {
   ambr_t used_ambr;
   subscriber_status_t subscriber_status;
   network_access_mode_t network_access_mode;
+  uint8_t num_reg_sub;
+  regional_subscription_t reg_sub[MAX_REGIONAL_SUB];
```

Comment:
do we really need to store this in ue_mm context? if so, then what about stateless support?